panel/chat/message.py

```diff
@@ -18,6 +18,7 @@
 )
 
 import param
+import pytz
```

Comment:
Generally I don't think you want to rely on transitive dependencies, i.e. Panel would break if `pandas` removed one day `pytz` from their direct dependencies.